src/Seksjon/Personalia.elm

```diff
@@ -132,27 +136,29 @@ oppdaterSamtaleSteg model samtaleSeksjon =
         }
 
 
-samtaleTilBoble : Samtale -> Snakkeboble
-samtaleTilBoble personaliaSeksjon =
+samtaleTilMeldingsLogg : Samtale -> List Melding
+samtaleTilMeldingsLogg personaliaSeksjon =
     case personaliaSeksjon of
         BekreftOriginal personalia ->
-            Robot
-                ("Jeg har hentet inn kontaktinformasjonen din. Den vil vises på CV-en."
-                    ++ "Det er viktig at informasjonen er riktig, slik at arbeidsgivere kan kontakte deg. "
-                    ++ "Fornavn: "
-                    ++ (Personalia.fornavn personalia |> Maybe.withDefault "-")
-                    ++ " Etternavn: "
-                    ++ (Personalia.etternavn personalia |> Maybe.withDefault "-")
-                )
+            [ Melding.spørsmål
+                [ "Da har du bekreftet personaliaen din"
+                ]
```

Comment:
Dette er vel feil? Jeg tror det som lå her fra før var riktig. For når man er i steget som heter `BekreftOriginal` så skal roboten si hva det originalet er, og så skal man få spørsmål om å man vi bekrefte eller endre.